.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,32 +1,33 @@
 ---
 name: Bug report
-about: Create a report to help us improve
+about: Report a bug or problem in Cirq or the project
 title: ''
 labels: 'kind/bug-report'
 assignees: ''
-
 ---
+## Describe the issue
 
-**Description of the issue**
 
-**How to reproduce the issue**
+## Explain how to reproduce the bug or problem
 
-```
+```text
```

Comment:
Nit - does it make a difference to use "text" here?  If we pre-fill the syntax, it would be more likely `python` than a `text`.

Reply:
You're right; will change.